cosi.proto

```diff
@@ -74,6 +74,59 @@ service Provisioner {
     rpc ProvisionerRevokeBucketAccess (ProvisionerRevokeBucketAccessRequest) returns (ProvisionerRevokeBucketAccessResponse);
```

Comment:
```PROTO
    rpc ProvisionerGetInfo (ProvisionerGetInfoRequest) returns (ProvisionerGetInfoResponse) {}
```

Should really move `ProvisionerGetInfo` out of `service Provisioner` in to a common `service Identity` unless you expect every mode of COSI deployment to always implement the Provisioner (e.g. even node instances).

Reply:
done. Check Line #55